confidential-identity/cli/scp/src/main.rs

```diff
@@ -299,22 +295,22 @@ fn process_create_claim_proof(cfg: CreateClaimProofInfo) {
         &raw_scope_claim.scope_did,
         &raw_scope_claim.investor_unique_id,
     );
-    let scope_claim_proof_data = build_scope_claim_proof_data(&cdd_claim, &scope_claim);
 
-    let pair = ProofKeyPair::from(scope_claim_proof_data);
-    let proof = pair.generate_id_match_proof(&message).to_bytes().to_vec();
+    let mut seed = [0u8; 32];
+    OsRng.fill_bytes(&mut seed);
+    let mut rng = StdRng::from_seed(seed);
 
-    let cdd_id = compute_cdd_id(&cdd_claim);
-    let scope_id = compute_scope_id(&scope_claim);
+    let proof = Investor::create_scope_claim_proof(&cdd_claim, &scope_claim, &mut rng);
 
     // => Investor makes {cdd_id, investor_did, scope_id, scope_did, proof} public knowledge.
     let packaged_proof = Proof {
-        cdd_id,
         investor_did: raw_cdd_claim.investor_did,
-        scope_id,
         scope_did: raw_scope_claim.scope_did,
+        scope_id: proof.scope_id,
+        cdd_id: proof.cdd_id,
         proof,
     };
+
```

Comment:
Did this to keep the output shape the same as it was. Otherwise, returning `proof` would have been enough since it has the same information.